bundle/src/main/java/com/adobe/acs/commons/http/JsonObjectResponseHandler.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2016 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.http;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.ResponseHandler;
+import org.apache.http.impl.client.BasicResponseHandler;
+import org.apache.sling.commons.json.JSONException;
+import org.apache.sling.commons.json.JSONObject;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+
+/**
+ * Converts response to a JSON Object.
+ */
+public class JsonObjectResponseHandler implements ResponseHandler<JSONObject> {
+
+    private BasicResponseHandler handler = new BasicResponseHandler();
+
+    @Override
+    public JSONObject handleResponse(HttpResponse httpResponse) throws
+            ClientProtocolException, IOException {
+        String json = handler.handleResponse(httpResponse);
+        if (json == null) {
```

Comment:
Good Q .. if the response body is blank, then handler.handleResponse(httpResponse) returns null.... if the response is blank, do we want an empty JSON obj or do we want to return null? guess thats up the contract of this API; i could go either way as long its documented.